--- origsrc/gnome-media-2.26.0/gst-mixer/src/Makefile.am	2009-03-04 17:54:53.000000000 -0600
+++ src/gnome-media-2.26.0/gst-mixer/src/Makefile.am	2009-04-19 02:52:32.343668700 -0500
@@ -6,7 +6,7 @@ endif
 
 AM_CPPFLAGS = \
 	$(GSTMIXER_CFLAGS)				\
-	-I../../sound-theme				\
+	-I$(top_srcdir)/sound-theme				\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\"	\
 	-DDATA_DIR=\""$(pkgdatadir)"\"			\
 	-DPIX_DIR=\""$(pkgdatadir)/pixmaps"\"
